build/Targets/VSL.Versions.targets

```diff
@@ -58,24 +48,24 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <BuildNumberSuffix Condition="('$(TF_BUILD_BUILDNUMBER)' != '')
-                     and ($(TF_BUILD_BUILDNUMBER.Split('_').Length) == 2)">$(TF_BUILD_BUILDNUMBER.Split('_')[1])</BuildNumberSuffix>
-    <BuildNumberPart1 Condition="'$(BuildNumberSuffix)' != ''">
-      $(BuildNumberSuffix.Split('.')[0])
+    <VSSemanticVersion>15.0.0</VSSemanticVersion>
+
+    <BuildNumberPart1 Condition="'$(BUILD_BUILDNUMBER)' != ''">
+      $(BUILD_BUILDNUMBER.Split('.')[0])
     </BuildNumberPart1>
-    <BuildNumberPart2 Condition="'$(BuildNumberSuffix)' != ''">
-      $(BuildNumberSuffix.Split('.')[1].PadLeft(2,'0'))
+    <BuildNumberPart2 Condition="'$(BUILD_BUILDNUMBER)' != ''">
+      $(BUILD_BUILDNUMBER.Split('.')[1].PadLeft(2,'0'))
     </BuildNumberPart2>
 
     <!-- Only set when building RTM with no dependencies on pre-release packages
-    <NuGetReleaseVersion>$(RoslynSemanticVersion)</NuGetReleaseVersion>
+    <*NuGetReleaseVersion>$(RoslynSemanticVersion)</*NuGetReleaseVersion>
     -->
```

Comment:
Took it offline. Since we will never release these packages these properties are unnecessary. Hence removing them